.gitignore

```diff
@@ -8,5 +8,7 @@ config/config.js
 coverage
 #MockData used for ProductOverview component
 MockData
+#AWS
+client/AWS
```

Comment:
Claire told me it was best to not include our AWS key anywhere in our project file. I think it's ok right now since you gitignore it, but it would be safer to just store it somewhere else.

Reply:
I agree, I saw Rob put the file in the computer and use the absolute path to get it. thanks